app/controllers/api/v1/responses_controller.rb

```diff
@@ -1,67 +1,76 @@
-class Api::V1::ResponsesController < ApplicationController
-  include ApplicationHelper
+module Api
+  module V1
+    class ResponsesController < ApplicationController
+      include ApplicationHelper
 
-  before_action :set_response, only: %i[show update]
-  before_action :require_user!, only: [:index, :show, :create, :update]
-  skip_before_action :verify_authenticity_token, only: [:response_flow_from_email]
+      before_action :set_response, only: %i[show update]
+      before_action :require_user!, only: [:index, :show, :create, :update]
 
-  def index
-    render json: ResponseSerializer.new(Response.all).serializable_hash
-  end
+      def index
+        render json: ResponseSerializer.new(Response.all).serializable_hash
+      end
 
-  def show
-    render json: ResponseSerializer.new(@response).serializable_hash.merge(additional_params)
-  end
+      def show
+        render json: ResponseSerializer.new(@response).serializable_hash.merge(additional_params)
+      end
 
-  def create
-    @response = current_user.responses.build(response_params)
+      def create
+        @response = current_user.responses.build(response_params)
```

Comment:
Does the @response variable in this method need to be global with a preceding `@`?

Reply:
@rogergraves it used in add_chosen_emotion method